documentation/docs/about/02-overview.md

```diff
@@ -1,2 +1,24 @@
 # Overview
 
+## Getting started
+
+In order to start using this template for your own project, go to [equinor/template-fastapi-react](https://github.com/equinor/template-fastapi-react) and click the `Use this template` button to create a copy. 
+
+Next, go-to the instructions on how-to [run locally](running/01-prerequisites.md) . 
+
+For setting up a development environment, go to the [development guide](../contribute/development-guide/01-setup.md). Next, to start coding and extending the template see the [coding section](../contribute/development-guide/coding/01-architecture.md).
```

Comment:
Good! By having these few sentences, the rest of the structure suddenly doesn't matter, because we now explain where stuff is. Nice.